app/controllers/admin/genres_controller.rb

```diff
@@ -1,22 +1,13 @@
 class Admin::GenresController < Admin::BaseController
 
-  def index
-    @genres = Genre.all
-  end
-
-  def new
-    @genre = Genre.new
-  end
-
   def create
-    @genre = Genre.new(genre_params)
-    if @genre.save
-      flash[:success] = "#{@genre.name} created successfully!"
-      redirect_to admin_genres_path
+    genre = Genre.new(genre_params)
+    if genre.save
+      flash[:success] = "#{genre.name} created successfully!"
     else
       flash[:failure] = "Genre could not be created. Please try again!"
-      render :new
     end
+    redirect_to genres_path
```

Comment:
good job moving this out of the if/else logic since both of those were going to redirect anyway